network-api/networkapi/buyersguide/templates/about/why_we_made.html

```diff
@@ -1,73 +1,119 @@
+      {% blocktrans trimmed %}
+      We want users of this guide to be able to share their opinion too. It’s important that companies, and other
+      consumers, see which products people think are safe, and which products people feel are a bit creepy. So we
+      created our Creep-O-Meter—a user rating on each product—to let folks give their opinion. Companies are paying
+      attention, so let them know if you think their product is creepy or not and if you’re likely to buy it.
+      {% endblocktrans %}
     </p>
   </section>
-  
+
   <section aria-labelledby="bigger-better">
-    <p id="bigger-better"><b>Bigger and better.</b></p>
+    <p id="bigger-better"><b>{% trans "Bigger and better." %}</b></p>
     <p class="body">
-      Our product list has grown to 76 connected products across six categories. You’ll notice a lot of research went into this guide. From knowing if the device uses a camera or microphone, or tracks your location or biometric data, to our section laying out what could happen if something goes wrong with a product—we’ve covered a lot of ground. Fortunately, we have great researchers and expertise here at Mozilla to dig into this so you don’t have to.  Our researchers combed through privacy policies, reached out to companies with questions about encryption and bug bounty programs, and sifted through product and app specifications. We’ve done our best to make your access to all this research easy to understand. 
+      {% blocktrans trimmed %}
+      Our product list has grown to 76 connected products across six categories. You’ll notice a lot of research went
+      into this guide. From knowing if the device uses a camera or microphone, or tracks your location or biometric data,
+      to our section laying out what could happen if something goes wrong with a product—we’ve covered a lot of ground.
+      Fortunately, we have great researchers and expertise here at Mozilla to dig into this so you don’t have to. Our
+      researchers combed through privacy policies, reached out to companies with questions about encryption and bug
+      bounty programs, and sifted through product and app specifications. We’ve done our best to make your access to all
+      this research easy to understand.
+      {% endblocktrans %}
     </p>
   </section>
 
   <div class="why-we-made-outro">
     <p class="body">
-      We hope you use and enjoy this guide to help you think about, shop for, and buy products from companies that show they value privacy and security. If you want to dig deeper, our friends on Mozilla’s Internet Health Report team have created a companion to our buyer’s guide that explains what can be done to make things better. It’s called <a href="https://foundation.mozilla.org/en/privacy-included/" target="_blank">*Privacy Included: Rethinking the Smart Home</a>.
+      {% blocktrans with link_args=' href="https://foundation.mozilla.org/privacy-included/" target="_blank"' trimmed %}
```

Comment:
here but also in a lot of other places, it'd be nicer to not have that space here, with `<a {{link_args}}>` in the text.

Reply:
I’m fine with changing that. The reason I moved it here, is because otherwise the link tag is not entirely highlighted in Pontoon and people can’t just click on it to copy the whole thing, they have to type `<a ` then click, then type `>`. Which introduces the risk they don’t add the space, and we would end up with `<ahref`. It hasn’t happened with the string below, though

![image](https://user-images.githubusercontent.com/1294206/75819654-faa5ce00-5d92-11ea-8adc-560f3f5908d2.png)

Reply:
(Updated)